todo_app/app.py

```diff
@@ -1,16 +1,19 @@
 from flask import Flask, render_template, request, url_for, redirect
 from todo_app.data.configs.flask_config import Config
-from todo_app.data.ViewModel import ViewModel
-import todo_app.data.session_items_trello as session
+from todo_app.data.view_model import ViewModel
+from todo_app.data.mongo_db_client import MongoDBClient
 import todo_app.data.configs.logger_settings as logger
 
 def create_app():
+    config = Config()
     app = Flask(__name__)
-    app.config.from_object(Config())
+    app.config.from_object(config)
+    mongo_db = MongoDBClient()
     
+
     @app.route('/')
     def index():
-        items = session.get_Items_and_Status()
+        items = mongo_db.get_items()
```

Comment:
Nice; that this is a clean switchover in this file is a good sign you  designed our interface well earlier!

Reply:
Thanks Jack